docs/get-started/quick-start.md

```diff
@@ -32,42 +32,51 @@ The following things are mandatory for both local development and production dep
 
 > NOTE: if you're unsure your AWS credentials are configured correctly, you can verify them by running the following command using [AWS CLI](https://aws.amazon.com/cli/): `aws sts get-caller-identity`. If you don't see the user info, take a look at this tutorial to [create an IAM user for programmatic usage](https://www.youtube.com/watch?v=tgb_MRVylWw).
 
-## 1. Install the CLI
+## 1. Create a new project
 
 ```
-npm -g install @webiny/cli
+npx create-webiny-project myProject --template templateName
 ```
 
-## 2. Create a new project
+> NOTE: The templateName can either be `full` (This will pull a package named `cwp-template-full` from npm template) or it can be a path to a local template ex. `../dev/template-to-use`
 
-```
-webiny create my-project
-cd my-project
-```
-
-## 3. Setup database connection
+## 2. Setup database connection
 
 Edit `.env.json` file in the root of the project and set the `MONGODB_SERVER` value. The values in this file are unique for your project, you do NOT need to change any other values.
 
 > In case you need any help creating a MongoDB cluster on MongoDB Atlas, please follow [this guide](/docs/other/guides/mongodb-atlas).
 
 > IMPORTANT: it's important to give the outside world access to your database because the database will be accessed from your cloud functions, thus you'll never have a fixed IP address. See the [Whitelist Your Connection IP Address](https://docs.atlas.mongodb.com/getting-started/#whitelist-your-connection-ip-address). Make sure you add a `0.0.0.0/0` entry.
 
-> The `MONGODB_SERVER` value should be in the format of a MongoDB connection string such as:
-> `mongodb+srv://{YOUR_USERNAME}:{YOUR_PASSWORD}@someclustername.mongodb.net`.
+Your `.env.json` file should look something like this after updating your `MONGODB_SERVER` and `MONGODB_NAME` parameters.
```

Comment:
Maybe it's just me, but I like putting `:` when I'm about to present a code snippet. So, I might put something like:
> After updating the `MONGODB_SERVER` and `MONGODB_NAME` variables, you might end up with something like the following:

Reply:
But in the end, leave what you feel is best.